libs/pcre/Config.in

```diff
@@ -0,0 +1,11 @@
+config PCRE_JIT_ENABLED
+	bool
+	depends on PACKAGE_libpcre && (arm || i386 || i686 || x86_64 || mips || mipsel || powerpc || sparc)
+	default y if (arm || i686 || x86_64)
```

Comment:
Why not enable it on other supported architectures?

Reply:
The reason was that I can only compile-test the other architectures since I do not have hardware to test on. I just did not want to break things as enabling JIT on unsupported architectures causes build-failures.

Reply:
ARM and x86 generally speaking have no size problems. This is fine.